tools/crm_resource_runtime.c

```diff
@@ -1351,8 +1351,8 @@ max_rsc_stop_timeout(pcmk_resource_t *rsc)
     meta = pcmk__unpack_action_meta(rsc, NULL, PCMK_ACTION_STOP, 0, config);
     if ((pcmk__scan_ll(g_hash_table_lookup(meta, PCMK_META_TIMEOUT),
                        &result_ll, -1LL) == pcmk_rc_ok)
-        && (result_ll >= 0) && (result_ll <= INT_MAX)) {
-        max_delay = (int) result_ll;
+        && (result_ll >= 0) && (result_ll <= UINT_MAX)) {
```

Comment:
I was thinking the same thing but stuck with the existing design. Not an issue in practice, but I'll change it to avoid getting annoyed every time we see it :)